pkgs/by-name/ea/eas-cli/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  yarnInstallHook,
+  nodejs,
+  jq,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "eas-cli";
+  version = "14.3.0";
```

Comment:
Per the previous comment, an update to 14.7.1 would be nice before merging this.